ruff.schema.json

```diff
@@ -0,0 +1,1150 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Options",
+  "type": "object",
+  "properties": {
+    "allowed-confusables": {
```

Comment:
Are these missing descriptions?

Reply:
Yes, there are a bunch of missing descriptions. I can add them :)

Reply:
It’s ok, I think I can update the macro to read from rustdoc, and then these comments can just be taking the “doc” piece from above and moving them into comments verbatim.

Reply:
The most helpful thing, in that light, would just be to take the existing doc= text from above and copying it as // docs for each attribute. But I can also do it, either way.